### PR TITLE
feat: Add "gno.land at GopherCon US 2024"

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

This PR adds the "gno.land at GopherCon US 2024" blog post, written by @VT-Cosmos.

### CODE REVIEW

File: posts/2024-08-12_gc-us24/README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+auto persistence, as well as its determinism and other features. The presentation
+also shared valuable lessons learned from GnoVM's development and implementation,
+offering insights into best practices and challenges overcome in the future work. 
+Be sure to make some time to watch [the recording](https://youtu.be/betUkghf_jo)!
+
+### Gno workshop
+
+Participants had the opportunity to dive deep into the development process, 
+guided by our expert engineer **[Dylan Boltz](https://github.com/deelawn)** who hosted a workshop on Community 
+Day: **The Building a decentralized app on Gno.land**. The workshop provided practical
```
Comment (Kouteki) [inline posts/2024-08-12_gc-us24/README.md:45]: Excess `The`

Comment (Kouteki) [reply posts/2024-08-12_gc-us24/README.md:45]: We should also use the lowercease `gno.land`

File: posts/2024-08-12_gc-us24/README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Be sure to make some time to watch [the recording](https://youtu.be/betUkghf_jo)!
+
+### Gno workshop
+
+Participants had the opportunity to dive deep into the development process, 
+guided by our expert engineer **[Dylan Boltz](https://github.com/deelawn)** who hosted a workshop on Community 
+Day: **The Building a decentralized app on Gno.land**. The workshop provided practical 
+insights and step-by-step guidance, empowering attendees to start building their
+own applications. If you missed our workshop or want to revisit the session, you 
+can find a recorded version of the workshop [here](https://www.youtube.com/watch?v=lwL2VyjaV-A).
```
Comment (Kouteki) [inline posts/2024-08-12_gc-us24/README.md:48]: This URL starts at 6:06

**Review by Kouteki — CHANGES_REQUESTED** [review]
1. I left a few comments that should be addressed
2. There's inconsistency in the title capitalization. E.g. `Highlights of our collaboration` VS `Jae's Talk`

File: posts/2024-08-12_gc-us24/README.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+developers.
+
+## Highlights of our collaboration
+
+### Jae’s Talk
+
+Jae Kwon, the founder of gno.land, gave a talk on “**Gno: Lessons in Building a
+Go Interpreter in Go.**”
+
+Jae’s  talk provided an in-depth overview of why this technology is gaining
```
Comment (Kouteki) [inline posts/2024-08-12_gc-us24/README.md:29]: Extra space

File: posts/2024-08-12_gc-us24/README.md (hunk 2 above)
Comment (leohhhn) [reply posts/2024-08-12_gc-us24/README.md:48]: Weird, starts normally for me. Can you check again, maybe in a incognito tab?

File: posts/2024-08-12_gc-us24/README.md (hunk 3 above)
Comment (leohhhn) [reply posts/2024-08-12_gc-us24/README.md]: [1457c52](https://github.com/gnolang/blog/pull/109/commits/1457c52b2198cd69d3e2ab241252ae7e210c2740)

File: posts/2024-08-12_gc-us24/README.md (hunk 1 above)
Comment (leohhhn) [reply posts/2024-08-12_gc-us24/README.md]: [da06a3f](https://github.com/gnolang/blog/pull/109/commits/da06a3f96fb56b816d46fa9ebf1747091161e80d) & [1cddd0d](https://github.com/gnolang/blog/pull/109/commits/1cddd0dd6534774f2973da831db974306a81f19f)

File: posts/2024-08-12_gc-us24/README.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+### The Challenge Series
+
+This year we had the privilege to participate in a collaborative partnership
+with the CodePro team to build the [challenge series](https://gophercon.challengeseries.org/). This collaboration was an
+opportunity for participants to learn how to interact with blockchain and 
+discover how realms (smart contracts) can be utilized for stateful applications
+without relying on explicitly managing a file system or database. This was an 
+opportunity to showcase gno.land’s features, like using a deployed contract as
+both an API and as an import for other contracts. For deeper insight take a look 
+at the recent blog written by Dylan Bolz on the collaboration with challenge
```
[the remark this answers is not in the document]
Comment (leohhhn) [reply posts/2024-08-12_gc-us24/README.md]: Removed the last sentence as that blog post is not out yet. Will be added later. [9da7d69](https://github.com/gnolang/blog/pull/109/commits/9da7d69154b89ae1f6a37c9179788d4b45769556)

File: posts/2024-08-12_gc-us24/README.md (hunk 2 above)
Comment (Kouteki) [reply posts/2024-08-12_gc-us24/README.md:47]: My bad, my cookie was messing with me. Thanks for pointing it out, Leon!

File: posts/2024-08-12_gc-us24/README.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+### The gno.land Booth
+One of the most rewarding feelings about attending these kinds of conferences
+is watching our booth quickly become a hub of activity, drawing a steady stream 
+of visitors intrigued by our project. We engaged with developers from various 
+backgrounds, answering a myriad of questions about Gno, gno.land, our company, 
+and the company’s [open employment opportunities](https://jobs.lever.co/allinbits).
+Having all of this direct human interaction was not only informative but also
+deeply insightful, providing us with valuable feedback and ideas. For more 
+information, check out our [landing page](https://gno.land/gophercon24).
```
[the remark this answers is not in the document]
Comment (leohhhn) [reply posts/2024-08-12_gc-us24/README.md]: Updated: [81c8f8b](https://github.com/gnolang/blog/pull/109/commits/81c8f8b7a93a7966e03477b977efdea51434377e)

**Review by Kouteki — APPROVED** [review]

**Pull request merged**